src/store/modules/map.store.js

```diff
@@ -129,5 +120,7 @@ export default {
         mapStoppedBeingDragged: (state) => (state.isBeingDragged = false),
         setPinnedLocation: (state, coordinates) => (state.pinnedLocation = coordinates),
         setDisplayedProjection: (state, projection) => (state.displayedProjection = projection),
+        displayLocationPopup: (state) => (state.displayLocationPopup = true),
+        hideLocationPopup: (state) => (state.displayLocationPopup = false),
```

Comment:
If you implement this as I've suggested in the store plugin, it would maybe be better to have this two distinct mutations moved to actions, and have only one mutation called `setDisplayLocationPopup` here